packages/dd-trace/src/plugins/tracing.js

```diff
@@ -12,6 +12,10 @@ class TracingPlugin extends Plugin {
     this.operation = this.constructor.operation
 
     this.addTraceSub('start', message => {
+      // TODO hasAsyncEnd is temporary until all plugins are converted to have asyncEnd
```

Comment:
Can a `WeakMap` be used instead? If that's possible then it would be possible to just always store the parent store and the additional property wouldn't be needed.